src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -16306,22 +16306,22 @@ public class C4 : I2, I3
             var comp = CreateCompilation(source, options: WithNonNullTypesTrue());
             comp.VerifyDiagnostics(
                 // (11,14): warning CS8645: 'I1<A>' is already listed in the interface list on type 'C1' with different nullability of reference types.
-                // public class C1 : I2, I1<A>
+                // public class C1 : I2, I1<A> 
```

Comment:
What's with all the trailing whitespace additions?

Reply:
There are trailing whitespaces in the source code above. Then copy&paste of the VerifyDiagnostics output. I'll clean up.

---
In reply to: [321007840](https://github.com/dotnet/roslyn/pull/38460#discussion_r321007840) [](ancestors = 321007840)